resources/css/index.css

```
@@ -330,3 +335,53 @@ div {
   display: flex;
   justify-content: center;
 }
+
+.fourth-section {
+  background: url("../images/background/section-5.jpg") no-repeat;
+  text-align: center;
+  padding: 100px 0;
+}
+
+.review {
+  display: inline-block;
+  background: rgba(0, 0, 0, 0.8);
```

```suggestion
  background-color: rgba(0, 0, 0, .8);
```
And check properties sorting

---

fixed

---

Disagree

---

fixed